component/libfuse/libfuse.go

```diff
@@ -246,6 +246,19 @@ func (lf *Libfuse) Validate(opt *LibfuseOptions) error {
 	return nil
 }
 
+func (lf *Libfuse) GenConfig() error {
```

Comment:
Rather than using string, let's use string builder.
Other way to write the same
func (lf *Libfuse) GenConfig() error {
	// Initialize the libfuse options based on current values
	options := LibfuseOptions{
		AttributeExpiration:     lf.attributeExpiration,
		EntryExpiration:         lf.entryExpiration,
		NegativeEntryExpiration: lf.negativeTimeout,
	}

	// If DirectIO is enabled, override expiration values
	if common.DirectIO {
		options.AttributeExpiration = 0
		options.EntryExpiration = 0
		options.NegativeEntryExpiration = 0
	}

	// Use a string builder to construct the YAML content
	var sb strings.Builder
	sb.WriteString("\nlibfuse:\n")

	// DirectIO check merged within value setting
	sb.WriteString(fmt.Sprintf(
		"  attribute-expiration-sec: %v\n  entry-expiration-sec: %v\n  negative-entry-expiration-sec: %v\n",
		options.AttributeExpiration, options.EntryExpiration, options.NegativeEntryExpiration))

	if common.DirectIO {
		sb.WriteString("  direct-io: true\n")
	}

	// Append generated YAML to the common configuration
	common.ConfigYaml += sb.String()

	return nil
}

Stringbuilder always perform better than string when we concatenate and build the result.

Reply:
done